src/app/components/Table/GoToFirstPageLink.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSearchParams, useHref } from 'react-router-dom'
+import Link from '@mui/material/Link'
+
+export const GoToFirstPageLink: FC = () => {
+  const { t } = useTranslation()
+  const [searchParams] = useSearchParams()
+  // We have no sure way to know the number of the page parameter to remove.
+  // The best we can do is to test the trivial option.
+  // If it's not there, then we can't really do anything.
```

Comment:
this should be responsibility of the page components

Reply:
The problem is that this component is rendered by the ErrorDisplayer, which is used by the ErrorBoundary, which is placed _outside_ the part that is doing the pagination. So it's impossible to pass the information down.

Theoretically it could be passed _up_ (inside the Error object), but we for various reasons we can't throw proper Errors (because that would trigger unwanted react behavior), just these strings, which can't carry information.

We could extend the string data structure somehow, but it's a pain...